docs/how-to/tls.rst

```diff
@@ -112,6 +112,8 @@ member.
    member. Any failure will result in subsequent communication errors for that
    service within the cluster.
 
+.. _certificates_lifecycle:
```

Comment:
Move the doc changes to a separate commit? Some of these changes will surely cause conflicts on backport, and it would be very nice to not have to deal with conflicts on the actual code changes.

Reply:
Yeah. I didn't slice this PR into commits very well.